File: TaskManagementSystemCleanArchitecture/Application/DTOs/Auth/LoginResponseDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.DTOs.Auth
{
    public class LoginResponseDto
    {
        public string Message { get; set; } = null!;
        public bool IsAuthenticated { get; set; }
        public string? Token { get; set; }
        public string? Username { get; set; }
       
    }
}

File: Frontend/TaskSystem/src/app/shared/models/Auth/login-response-dto.ts
export interface LoginResponseDto {
    message: string;
    isAuthenticated: boolean;
    token: string;
    username: string;
    
    role: string;
      /* public string Message { get; set; } = null!;
 public bool IsAuthenticated { get; set; }
 public string? Token { get; set; }
 public string? Username { get; set; }
 public string role { get; set; } */
}
